starters/apps/basic/src/root.tsx

```diff
@@ -20,7 +21,7 @@ export default component$(() => {
     <QwikCityProvider>
       <head>
         <meta charSet="utf-8" />
-        <link rel="manifest" href="/manifest.json" />
+        <link rel="manifest" href={`${BASE_URL}manifest.json`} />
```

Comment:
does BASE_URL end with a /?

Reply:
@JerryWu1234 one question then I can merge

Reply:
I'm going to merge and we can figure out later (at least we have time before we cut a release)

Reply:
It will be better to add / here instead of assuming BASE_URL ends with /, because even if it has a // will be treated as /